src/base/bittorrent/sessionimpl.cpp

```diff
@@ -4084,7 +4090,7 @@ void SessionImpl::setQueueingSystemEnabled(const bool enabled)
         configureDeferred();
 
         if (enabled)
-            saveTorrentsQueue();
+            m_torrentsQueueChanged = true;
```

Comment:
Doesn't coding style demand you to add curly brackets to `if else` body?

Reply:
> Doesn't coding style demand you to add curly brackets to `if else` bodies?

Each body is single line, so not.